zerver/lib/email_mirror.py

```diff
@@ -164,14 +163,11 @@ class ZulipEmailForwardUserError(ZulipEmailForwardError):
 def send_zulip(sender: str, stream: Stream, topic: str, content: str) -> None:
     trunc_topic = truncate_topic(topic)
     if len(trunc_topic) < len(topic):
-        pdlte = datetime.datetime.today()
-        pdgt = pdlte - datetime.timedelta(days = 7)
         if not (Message.objects.filter(
                 subject = trunc_topic,
-                recipient = get_stream_recipient(stream.id),
-                date_sent__lte = pdlte,
-                date_sent__gt = pdgt).exists()):
-            content = 'Subject: ' + topic + '\n\n' + content
+                recipient = stream.recipient_id,
```

Comment:
Ah, I didn't know that actually works - but I think it's better to use ``recipient_id`` if we're filtering by the id, as opposed to a ``Recipient`` object - even though Django is able to handle ``recipient=<integer>`` expression.